### PR TITLE
Bug fixes

### PR DESCRIPTION
**Pull request by AlanRoth**
Fixed Q bug
Fixed zoom bug
Refactor word handler to use blocks, implemented a filter to pick out the matching blocks to the valid word

### CODE REVIEW

File: Assets/Scripts/GlobalValues/Alphabet.cs [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 public class Alphabet
 {
-    public static char[] alphabet = "TEST".ToCharArray();
+    public static char[] alphabet = "ABCDEFGHIJKLMNOPQRSTVWXYZ".ToCharArray();
```
Comment (MattBinnie) [inline Assets/Scripts/GlobalValues/Alphabet.cs:7]: we both made this change - i will solve the conflict after merging yours

File: Assets/Scripts/WordHandler/WordHandler.cs [hunk 2]
```diff
@@ -6,28 +6,32 @@
...
+                return blocks;
+            }
+
             for(int i = 0; i < blocks.Count(); i++)
             {
 
-                lettersOnRow.Add(char.Parse(blocks.ElementAt(i).GetComponent<Block>().GetValue()));
-                blocksToRemove.Add(blocks.ElementAt(i));
+                blocksOnRow.Add(blocks.ElementAt(i));
+                //blocksToRemove.Add(blocks.ElementAt(i));
```
Comment (MattBinnie) [inline Assets/Scripts/WordHandler/WordHandler.cs:34]: needed?

Comment (AlanRoth) [reply Assets/Scripts/WordHandler/WordHandler.cs:34]: good spot

File: Assets/Scripts/GlobalValues/Alphabet.cs (hunk 1 above)
Comment (AlanRoth) [reply Assets/Scripts/GlobalValues/Alphabet.cs:7]: might be ok if its exactly the same

**Pull request merged**